R/clear_stash.R

```diff
@@ -1,14 +1,16 @@
 #' Clear the stash
 #'
 #' Clears the hidden '.mustashe' directory.
+#' 
+#' @param verbose Whether to print action statements (default TRUE).
 #'
 #' @return Returns \code{NULL} (invisibly).
 #'
 #' @examples
 #' clear_stash()
 #' @export clear_stash
-clear_stash <- function() {
-  message("Clearing stash.")
+clear_stash <- function(verbose=TRUE) {
+  if(verbose) message("Clearing stash.")
```

Comment:
It's not the end of the world and you can leave it as is if you want, but I do prefer a space between the `if` and open ing parentheses and braces around the code block of an if statement: 

```r
if (verbose) { message("Clearing stash.") }
```

If you do make this change, can you update the other places, too? I'm thinking of adding a GH Action for styling the code in the future, so it really isn't a problem if you don't change this.